packages/sdks/e2e/react-native-76-fabric/app/[...slug].tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  Content,
+  _processContentResult,
+  fetchOneEntry,
+} from '@builder.io/sdk-react-native';
+import { getProps } from '@sdk/tests';
+import { useLocalSearchParams } from 'expo-router';
+import { useEffect, useState } from 'react';
+import { Text, View } from 'react-native';
+
+export default function HomeScreen() {
+  const [props, setProps] = useState<any>(undefined);
```

Comment:
Question: why didnt we use `import type { BuilderContent } from '@builder.io/sdk-react-native';` here?